src/ocean_data_assim/MOM_oda_driver.F90

```diff
@@ -678,8 +685,8 @@ subroutine apply_oda_tracer_increments(dt, Time_end, G, GV, tv, h, CS)
                                                     !! tendency [C T-1 -> degC s-1]
   real, dimension(SZI_(G),SZJ_(G),SZK_(G)) :: S_inc !< an adjustment to the salinity
                                                     !! tendency [S T-1 -> ppt s-1]
-  real, dimension(SZI_(G),SZJ_(G),SZK_(CS%Grid)) :: T !< The updated temperature [C ~> degC]
-  real, dimension(SZI_(G),SZJ_(G),SZK_(CS%Grid)) :: S !< The updated salinity [S ~> ppt]
+  real, dimension(SZI_(G),SZJ_(G),SZK_(CS%Grid)) :: T !< The temperature tendency adjustment from DA [C T-1 ~> degC s-1]
```

Comment:
The code is now correctly documented, but for clarity would it make sense to rename these local variables from `T` and `S` to `T_tend` and `S_tend`?